packages/components/src/BulkActionBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { ActionBar } from '@adobe/react-spectrum';
+import type { StyleProps } from '@react-types/shared';
+import commonStyles from './SpectrumComponent.module.scss';
+
+export interface BulkActionBarProps {
+  styleProps?: StyleProps;
+  selectedItemCount: 'all' | number;
+  onClearSelection: () => void;
+}
+
+export function BulkActionBar({
```

Comment:
I think this is from your other PR? Shouldn't be part of this PR...

Reply:
rebase took care of this